packages/services/f-braze-adapter/src/services/_tests/BrazeDispatcher.test.js

```diff
@@ -152,112 +160,114 @@ describe('instantiation', () => {
 
                     // Assert
                     expect(appboySDK.subscribeToInAppMessage).toHaveBeenCalledAfter(appboySDK.openSession);
                 });
 
-                it('should set the userId following callback subscription', async () => {
-                    // Act
-                    await brazeDispatcher.configure(enabledComponentParameters);
+                it('should set the userId following callback subscription', () => {
+                    // Arrange & Act
+                    const dispatcher = new BrazeDispatcher(enabledComponentParameters);
 
                     // Assert
                     expect(appboySDK.changeUser).toHaveBeenCalledAfter(appboySDK.requestContentCardsRefresh);
                 });
 
-                it('should pass through the userId along with a callback', async () => {
-                    // Act
-                    await brazeDispatcher.configure(enabledComponentParameters);
+                it('should pass through the userId along with a callback', () => {
+                    // Arrange & Act
+                    const dispatcher = new BrazeDispatcher(enabledComponentParameters);
 
                     // Assert
                     expect(appboySDK.changeUser).toHaveBeenCalledWith(userId, expect.any(Function));
                 });
             });
 
-            it.each([
-                [true],
-                [false]
-            ])('should reject being called a second time with different parameters', async appboyInitialised => {
-                // Arrange
-                expect.assertions(1);
-                isAppboyInitialised.mockReturnValue(appboyInitialised);
-                await brazeDispatcher.configure(enabledComponentParameters);
-
-                // Act
-                try {
-                    await brazeDispatcher.configure({
-                        ...enabledComponentParameters,
-                        userId: '__INVALID_USER_ID__'
-                    });
-                } catch (error) {
-                    // Assert
-                    expect(error.message).toBe('Attempt to reinitialise appboy with different parameters');
-                }
-            });
-
-            it.each([
-                [true],
-                [false]
-            ])('should return the same promise if called a second time with similar parameters', async appboyInitialised => {
-                // Arrange
-                isAppboyInitialised.mockReturnValue(appboyInitialised);
-                const promiseA = await brazeDispatcher.configure(enabledComponentParameters);
-
-                // Act
-                const promiseB = await brazeDispatcher.configure(enabledComponentParameters);
-
-                // Assert
-                expect(promiseA).toBe(promiseB);
-            });
-
-            it('should push an event to datalayer when userId set following initialisation', async () => {
+            // it.each([
```

Comment:
Are these to be reinstated at some point? If not, I'd prefer if these were just deleted, rather than commenting out, and if so, please add a comment explaining what is happening with them.

Reply:
correct they can be removed